caikit/runtime/work_management/call_aborter.py

```diff
@@ -26,11 +26,12 @@
 
 # Local
 from caikit.runtime.types.caikit_runtime_exception import CaikitRuntimeException
+from caikit.runtime.work_management.abortable_action import ActionAborter
 
 log = alog.use_channel("CALL-ABORTER")
 
 
-class CallAborter:
+class CallAborter(ActionAborter):
```

Comment:
Maybe we should rename this class to `RPCAborter`? Since it specifically uses a grpc context